.github/workflows/definitelyTyped.yml

```diff
@@ -0,0 +1,55 @@
+name: DefinitelyTyped Tests
+
+on:
+  push:
+    branches:
+      - 'feature/definitelytype'
+    paths:
+      - 'src/**'
+      - 'definitelyTyped/**'
+
+jobs:
+  performance:
```

Comment:
Could you rename the job to definitelyType.. Basically this name is useful in GitHub to understand which pipeline is.

Let's say for example we want to make this pipeline required for a specific branch, with that name we can select it from the list

Reply:
This pipeline is just to run the definitely typed tests on the master branch, once I put the cron there is no branch to specify I think

Reply:
I did not explain myself correctly.

jobs:
   performance -> definitelyTyped

Reply:
Ah sorry